README2.md

```
@@ -0,0 +1,30 @@
+
+2. Run Spring Boot
+   mvn spring-boot:run
+
+3. Open the HTML page
+   http://localhost:8080/quotes
+
+4. Test the REST API
+   GET http://localhost:8080/api/quotes
+
+The app loads movie quotes from a database and displays them both in API- and webformat. 
+### JSON = _http://localhost:8080/api/quotes_
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)